metabase/metabase
# Fix scrollbar interaction in the custom expression suggestion list

Closes https://github.com/metabase/metabase/issues/41165

### Description
The underlying issue is that AceEditor has an `onBlur` event handler, triggered once the user clicks out of the `textarea` (in this case, onto the `ExpressionEditorSuggestions` scrollbar).

### How to verify
1. Start a new question based on the Orders table
2. Click + to add a new column
3. Type [ to show suggestions list
4. Try to press & hold the vertical scrollbar to scroll the list
5. Expression Suggestions should be scrollable and remain visible 

### Demo

https://github.com/metabase/metabase/assets/59742/fe37c5ab-fd6c-4afa-9cba-978d4f477b9e



### Checklist

- [x] Writing a test to verify this would be tricky. It's hard to mock, and Cypress has an issue with clicking on the scrollbar
